Url Shortening simple web application

OmarManzoor opened the pull request:
All the files committed and need a review

verdanmahmood commented on url_shortening/my_shortener/models.py:
```diff
@@ -0,0 +1,12 @@
+
+class UrlMap(models.Model):
+    word = models.ForeignKey(Wordlist, blank=True, null=True)
+    url_actual = models.CharField(max_length=300)
```
@OmarManzoor 
you should use the Django URL field here.

verdanmahmood commented:
@OmarManzoor you missed the Test Cases here. (as stated in the doc)

OmarManzoor commented:
Made corrections and changes in the code by using a datetime for Words in the model to keep track of when that word was entered and updated. Also used a URL field for keeping track of the original url. Also made class based views and added some tests
